### ISSUE TITLE
README.md file confusing

### ISSUE THREAD

**Issue by HowardCohl** [ContemporaryArtwork/M-to-m#6]
Hi Jacob,

I was reading your README.md file in the M-to-m task, and upon careful reading, it doesn't make complete sense.  It is not clear in the last sentence which program you are talking about. 

Could you be more specific in the entire README.md file about what programs there are, what they are called, and what they do?

I think you should follow the guidelines presented in http://stackoverflow.com/questions/2304863/how-to-write-a-good-readme

**Comment by HowardCohl** [comment]
This might be related to https://github.com/ContemporaryArtwork/M-to-m/issues/5

**Comment by HowardCohl** [comment]
@ContemporaryArtwork 

You should mention something about the directory where the source is located in the README.me file.
I didn't see the input .tex file in that directory. Is this ok?

**Comment by ContemporaryArtwork** [comment]
I forgot to ask, did you use git to clone the repository? Because there should be a tex file in main/resources and test/resources.

**Comment by HowardCohl** [comment]
Yes I used git to clone the repository. I think I mentioned that here:

https://github.com/ContemporaryArtwork/M-to-m/issues/5#issuecomment-162597964

Ok, I just looked yes, they are there. Do I always have to put my input files on those directories?  Can I put them in other directories? Do I have to put the same .tex file in both directories? How do I call the program? Where does the output appear?  I think all of this should be in the README.md file.What do you think @physikerwelt ?

**Comment by physikerwelt** [comment]
I agree

**Comment by HowardCohl** [comment]
@physikerwelt 

Does this mean you would like to see all this information in the README.md file?
Can you summarize the full extent of what information should be provided there?

* location of and type of source files
* location and type of input files
* location and type of output files
* calling sequence for code if run at command line
* method for running code if another method is preferred as well over running at the command line

anything else?
